fastlane_core/lib/fastlane_core/configuration/config_item.rb

```diff
@@ -100,6 +106,7 @@ def initialize(key: nil, env_name: nil, description: nil, short_option: nil, def
       @deprecated = deprecated
       @sensitive = sensitive
       @allow_shell_conversion = (type == :shell_string)
+      @display_in_shell = display_in_shell
```

Comment:
@KrauseFx It was pretty ugly to combine into one line so I changed the default value in the method definition to `true` 😇  I think this was the cleanest way of doing it?